app/containers/AddressChooserModalContainer.tsx

```diff
@@ -0,0 +1,170 @@
+import { debounce } from "lodash";
+import * as React from "react";
+
+import { AddressChooserModalComponent } from "../components/AddressChooserModalComponent";
+import config from "../config";
+import { ledgerInstance, web3Instance } from "../web3/web3Provider";
+
+const NUMBER_OF_ADDRESSES = 5;
+
+export interface IAddresses {
```

Comment:
You could use existing dictionary type (because it is a dictionary xD) and parametrize it. Hit me if you have problems with it.